frontend/public/locales/en/status.json

```
@@ -90,6 +93,8 @@
     "page-title": "Status Checker",
     "status-heading": "Status",
     "check-another": "Check another status",
+    "exit-btn": "Exit status checker",
+    "exit-link": "https://www.canada.ca/en/services/benefits/dental/dental-care-plan.html",
```

Based on Figma, it must be:

```suggestion
    "exit-link": "https://www.canada.ca/en/services/benefits/dental/dental-care-plan/apply.html#status",
```